corehq/apps/reports/views.py

```diff
@@ -1937,20 +1937,26 @@ def archive_form(request, domain, instance_id):
 
 
 def _get_cases_with_forms_message(domain, cases_with_other_forms, case_id_from_request):
-    def _get_case_link(case_id, name):
-        if case_id == case_id_from_request:
-            return _("%(case_name)s (this case)") % {'case_name': name}
-        else:
-            return '<a href="{}#!history">{}</a>'.format(reverse('case_data', args=[domain, case_id]), name)
+    def _get_all_case_links():
+        all_case_links = []
+        for case_id, case_name in cases_with_other_forms.items():
+            if case_id == case_id_from_request:
+                all_case_links.append([(
+                    format_html(_("{}s (this case)"), case_name)
```

Comment:
I see that the previous format string was: "%(case_name)s (this case)", where `%` the trailing `s` after case_name indicates it should be parsed as a string. If you want to replicate that using [format string syntax](https://docs.python.org/3.4/library/string.html#format-string-syntax), you'd do `{case_name!s}` or `{!s}`. That said, I've been shying away from this and just using `{}`, especially when the template string is what our translators are expected to use -- I don't want to confuse them with python details any more than necessary. I think explicit string conversion is unnecessary.

```suggestion
                    format_html(_("{} (this case)"), case_name)
```